docker/docker-compose.yml

```diff
@@ -29,16 +29,20 @@ services:
     image: swaggerapi/swagger-ui
     container_name: "swagger-ui"
     ports:
-      - "8081:8080"
+      - "8081:8080" # debug only; use reverse-proxy http://localhost/swagger/
     restart: 'always'
     volumes:
-      - ./swagger/nexus-swagger.json:/usr/share/nginx/html/api/nexus-swagger.json
-      - ./swagger/portal-swagger.json:/usr/share/nginx/html/api/portal-swagger.json
+      - ./swagger/nexus-swagger.json:/usr/share/nginx/html/apiapi/nexus-swagger.json
+      - ./swagger/portal-swagger.json:/usr/share/nginx/html/apiapi/portal-swagger.json
     environment:
-      URLS: "[{ url: \"./api/nexus-swagger.json\", name: \"Nexus\"   },
-              { url: \"./api/portal-swagger.json\", name: \"Portal\" },
+      - "URLS=[{ url: \"./apiapi/nexus-swagger.json\", name: \"Nexus\"   },
+              { url: \"./apiapi/portal-swagger.json\", name: \"Portal\" },
```

Comment:
Is apiapi a typo?

Reply:
This is a workaround vhost rule in apache